samples/client/petstore/jaxrs-cxf/src/gen/java/io/swagger/api/PetApi.java

```diff
@@ -15,64 +15,62 @@
 
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import io.swagger.jaxrs.PATCH;
 
 @Path("/")
 @Api(value = "/", description = "")
-@Consumes(MediaType.APPLICATION_JSON)
-@Produces(MediaType.APPLICATION_JSON)
 public interface PetApi  {
 
     @POST
     @Path("/pet")
     @Consumes({ "application/json", "application/xml" })
     @Produces({ "application/xml", "application/json" })
     @ApiOperation(value = "Add a new pet to the store", tags={  })
-    public void  addPet(Pet body);
+    public void addPet(Pet body);
 
     @DELETE
     @Path("/pet/{petId}")
     @Produces({ "application/xml", "application/json" })
     @ApiOperation(value = "Deletes a pet", tags={  })
-    public void  deletePet(@PathParam("petId") Long petId, @HeaderParam("api_key") String apiKey);
+    public void deletePet(@PathParam("petId") Long petId, @HeaderParam("api_key") String apiKey);
 
     @GET
     @Path("/pet/findByStatus")
     @Produces({ "application/xml", "application/json" })
     @ApiOperation(value = "Finds Pets by status", tags={  })
-    public List<Pet>  findPetsByStatus(@QueryParam("status")List<String> status);
+    public List<List<Pet>> findPetsByStatus(@QueryParam("status")List<String> status);
```

Comment:
This does not seem to be correct as `findPetsByStatus` should return a list of Pet, not a list of list of Pet.